vertical-pod-autoscaler/pkg/admission-controller/certs.go

```diff
@@ -92,6 +119,36 @@ func (cr *certReloader) load() error {
 	return nil
 }
 
+func (cr *certReloader) reloadWebhookCA() error {
```

Comment:
Could you try add a test for this?

Reply:
Wrote some in https://github.com/kubernetes/autoscaler/pull/7454/commits/4a09aa3ff37a468a4c1de9547f2aa221c6727d67